.github/merge-when-green.yml

```diff
@@ -0,0 +1,13 @@
+# See https://github.com/phstc/probot-merge-when-green for documentation of configuration options.
+
+# requiredChecks:
+#   - circleci
+#   - travis-ci
+# requiredStatuses:
+#   - jenkins
+
+requireApprovalFromRequestedReviewers: true
```

Comment:
Are you sure this will do what you want w/o travis-ci configured? Looks like it might just merge as soon as we approve.

Reply:
Oh, I see: 

> By default, Merge when green will wait for branch protection to pass and also require circleci and travis-ci before attempting to merge.

Now I'm wondering if it will hang out forever waiting for the `circleci` checks we don't run. Let's find out!

Reply:
I would have assumed it just waits for all checks to pass, but if you have to opt-in… seems to be doing the right thing:

![image](https://user-images.githubusercontent.com/101754/67032052-ad68de80-f0c7-11e9-8d97-5d58b956b78b.png)